packages/connectivity/src/scp-cf/destination/destination-from-service.ts

```diff
@@ -434,10 +485,14 @@ Possible alternatives for such technical user authentication are BasicAuthentica
     if (!this.subscriberToken) {
       throw new Error('Try to get subscriber token but value is undefined.');
     }
-    return (
-      this.subscriberToken.userJwt?.decoded ||
-      this.subscriberToken.serviceJwt.decoded
-    );
+    switch (this.subscriberToken.type) {
+      case 'custom':
+        return this.subscriberToken.userJwt.decoded;
+      case 'iss':
```

Comment:
[req] I actually wanted to comment on line 483. Since the new `IssToken` interface specifically set `userJwt` to `undefined`, maybe we can remove this line of comment.